rj_gameplay/rj_gameplay/tactic/pass_with_seekers.py

```diff
@@ -0,0 +1,127 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from typing import Dict, Generic, List, Optional, Tuple, Type, TypeVar, Callable
+
+import stp.action as action
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+
+import rj_gameplay.eval
+import rj_gameplay.skill as skills
+import rj_gameplay.tactic.pass_seek as pass_seek
+from rj_gameplay.skill import temp_pivot_skill, pivot_kick, receive
+import stp.skill as skill
+import numpy as np
+
+
+class pass_cost(role.CostFn):
+    """
+    A cost function for how to choose a robot to pass to
+    TODO: Implement a better cost function
+    """
+    def __init__(self, target_point:Optional[np.ndarray] = None):
+        self.target_point = target_point
+
+    def __call__(
+        self,
+        robot: rc.Robot,
+        prev_result: Optional["RoleResult"],
+        world_state: rc.WorldState,
+    ) -> float:
+    
+        if robot.id == 7:
+            return 0.0
+        return 1.0
+
+class passer_cost(role.CostFn):
+    """
+    A cost function for how to choose a robot that will pass
+    TODO: Implement a better cost function
+    """
+    def __call__(self,
+                robot:rc.Robot,
+                prev_result:Optional["RoleResult"],
+                world_state:rc.WorldState) -> float:
+        if robot.has_ball_sense:
+            return 0
+        else:
+            if robot.id == 1:
+                return 0
+            else:
+                return 1
+
+
+
+class Pass(tactic.ITactic):
+    """
```

Comment:
I'm a little confused about the difference between this file and the `tactic/pass_seek.py` file. Could you explain?

Reply:
Ah, this was a mistake I thought I wanted 2 different types of passing but just pass_tactic is good.